crates/dbsp/src/storage/buffer_cache/cache.rs

```diff
@@ -275,65 +281,73 @@ where
     E: CacheEntry,
 {
     fn create(&self) -> Result<FileHandle, StorageError> {
-        self.backend.create()
+        self.backend.get_or(self.new_backend_fn).create()
```

Comment:
`self.backend.get_or(self.new_backend_fn)` is a common pattern, should there be a helper function?